pyproject.toml

```diff
@@ -16,55 +15,67 @@ classifiers = [
   "Topic :: Software Development",
   "Intended Audience :: Developers",
   "Typing :: Typed",
+  "License :: OSI Approved :: MIT License",
+]
+requires-python = ">=3.11"
```

Comment:
Are you intentionally allowing 3.13 now?

Reply:
yep, one of the maintainer of uv recommends not setting an upper bound. I don't have a link right now to the comment but it's on their GitHub

Reply:
found it: https://github.com/astral-sh/uv/issues/6783#issuecomment-2316648253

Reply:
More asking generally and out of curiosity now: Is uv greedy then and tries to set up the project with the latest Python stable version when I clone an existing project for the first time? Or can uv be configured globally to try to use the latest Python stable version up to and including e.g. 3.12?

Reply:
AFAIK you can do `uv python pin 3.12`. I haven't tested it thoroughly so far